db/LousListScript.md

```diff
@@ -0,0 +1,18 @@
+# Lou's List Script
+[UVA Lou's List](louslist.org) links individual courses to tCF course pages, and it requires a table mapping course IDs to the IDs in our URLs (mnemonic, number, database id). They are then fetched at https://thecourseforum.com/course/[course_id].
+
+Run the following commands to generate a table for Lou's List using our database:
+
+0. Make sure you have your database set up correctly. It is likely that you ran `docker-compose down` to start with a clean slate.
+
+1. Open the psql terminal in the tCF database Docker container with this command*:
+    ```
+    docker exec -it tcf_db psql -U tcf_django tcf_db -f app/export_mapping_table.sql
```

Comment:
Connected to above; I've tested and it works without the `-it` flag (as it's just running one thing)

Reply:
Removing `-it` also makes it work on Git Bash on Windows!

Reply:
Fixed in https://github.com/thecourseforum/theCourseForum2/pull/237/commits/6d71c9265e5ae72ff4aebc7cbad263c729e2b81d. Thanks for testing it out as I couldn't test it because I don't have it installed.

Reply:
@alex-shen1 What if you remove just the `-t` flag? I was running another command, and it seems like `-i` is necessary in that case.

Reply:
Yeah, it works if you do `-i` or `-t`, but breaks only if you do `-it`. Weird; I think it's something with the terminal emulator Git Bash uses on Windows (weirdly enough, if you run Git Bash from Pycharm `-it` works fine)